src/video_core/engines/maxwell_3d.h

```diff
@@ -1137,6 +1137,11 @@ class Maxwell3D final {
 private:
     void InitializeRegisterDefaults();
 
+    void InitDirtyRegsMask();
+    void InitExecRegsMask();
+    void CheckDirty(const u32 method);
```

Comment:
```suggestion
    void CheckDirty(u32 method);
```

value-wise const only has a meaningful use in the definition, not the declaration of a function.